ui/ssb-msg.js

```diff
@@ -88,6 +104,31 @@ Vue.component('ssb-msg', {
         else
           this.msg = { key: this.msg.key, value: msgValue }
       })
+    },
+    react: function(emoji) {
+      if(emoji == 'Unlike') {
+	this.myReactions = [];
+      } else {
+        this.myReactions.push({ expression: emoji });
+      }
+      var reactTo = this.msg.key;
+      var message = {
+        type: 'vote',
+	vote: {
+	  link: reactTo,
+	  value: 1,
```

Comment:
For unlike the value needs to be [0](https://github.com/ssbc/patchwork/blob/ae29bd7f91e16deac4bd88b099175fa85a419815/lib/depject/message/html/actions.js#L58)